recipe/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "curl-impersonate" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lexiforest/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: be2494020a26378ceb888c2198e3cc1b8dee1150e58e5f432aa3bb92659d8e17
+  patches:
+    - patches/0001-unix-makefile-fix.patch  # [unix]
+    - patches/0002-windows-unvendor.path  # [win]
+
```

Noting https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#source-from-git we can get `conda-build` to download and verify the specific git commits which means the downloads aren't lost in build scripts:

```suggestion
source:
  - url: https://github.com/lexiforest/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
    sha256: be2494020a26378ceb888c2198e3cc1b8dee1150e58e5f432aa3bb92659d8e17
    patches:
      - patches/0001-unix-makefile-fix.patch  # [unix]
      - patches/0002-windows-unvendor.path  # [win]
  - git_url: https://github.com/org/repo
    git_rev: commitID/tag/??
    folder: deps/xxx
    sha256: abc123
...
```